### PR TITLE
Initial version of the edr-pydantic package

### PR DESCRIPTION
**Pull request by rosinaderks**

### CODE REVIEW

File: src/edr_pydantic/link.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    rel: str
+    # TODO According to A.21 & A.23 all links shall include type
+    type: Optional[str] = None
+    title: Optional[str] = None
+    length: Optional[int] = None
+    templated: Optional[bool] = None
+    variables: Optional[Union[Variables, CubeVariables, CorridorVariables, ItemVariables, RadiusVariables]] = None
+
+
+# For EDRQueryLink the variables element is required compared to the Link object
```
Comment (lukas-phaf) [inline src/edr_pydantic/link.py:24]:
```suggestion
# For EDRQueryLink the variables element is required unlike in other Link objects
```

File: src/edr_pydantic/link.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+class EDRQueryLink(Link, extra="allow"):
+    variables: Union[Variables, CubeVariables, CorridorVariables, ItemVariables, RadiusVariables]
```
Comment (lukas-phaf) [inline src/edr_pydantic/link.py:26]: This actually works? Nice.

File: src/edr_pydantic/my_base_model.py
Comment (lukas-phaf) [inline src/edr_pydantic/my_base_model.py:1]: Rename this file to `base_model.py`

File: src/edr_pydantic/my_base_model.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel as PydanticBaseModel
+from pydantic import ConfigDict
+
+
+class EDRBaseModel(PydanticBaseModel):
+    model_config = ConfigDict(
+        str_strip_whitespace=True,
+        extra="forbid",
```
Comment (lukas-phaf) [inline src/edr_pydantic/my_base_model.py:8]: Also add `str_min_length=1` like in `covjson-pydantic`, or does that break stuff?

File: src/edr_pydantic/my_base_model.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel as PydanticBaseModel
+from pydantic import ConfigDict
+
+
+class EDRBaseModel(PydanticBaseModel):
```
Comment (lukas-phaf) [inline src/edr_pydantic/my_base_model.py:5]: Rename to ``EdrBaseModel`?

File: src/edr_pydantic/reference_system.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from .my_base_model import EDRBaseModel
+
+
+class CRS(EDRBaseModel):
+    name: str  # MAY EPSG code
+    wkt: str  # SHALL Well Known Text
```
Comment (lukas-phaf) [inline src/edr_pydantic/reference_system.py:6]: Keep the comments like this?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+Apache License, Version 2.0
+
+## Authors
+
+Members of the KNMI Data Platform team. Reachable at opendata@knmi.nl.
+
+## Copyright
+
+Koninklijk Nederlands Meteorologisch Instituut (KNMI)
```
[the remark this answers is not in the document]
Comment (PaulVanSchayck) [reply README.md:163]: Regarding AUTHORS. I don't mind listing my name in the README. The reason I put it to "Members of the KNMI Data Platform team." is so no one felt left out :P 

Regarding Copyright. Technically, the Copyright is with our employer, as you probably know. See also the LICENSE file. If someone outside KNMI contributes to the project, the copyright becomes joint with them, and we should add them to the list.

File: src/edr_pydantic/my_base_model.py (hunk 3 above)
Comment (PaulVanSchayck) [reply src/edr_pydantic/my_base_model.py:8]: It's in there, isn't it?

File: src/edr_pydantic/reference_system.py (hunk 5 above)
Comment (PaulVanSchayck) [reply src/edr_pydantic/reference_system.py]: Improved. I've a few question still about this object. Trying to resolve those with @rosina-derks

Comment (rosinaderks) [reply src/edr_pydantic/reference_system.py]: Discussed with @lukas-phaf and @PaulVanSchayck that this object is not required in the spec. For this reason, the object will be removed.

**Review by lukas-phaf — APPROVED** [review]

**Pull request merged**